internal/cli/performanceadvisor/namespaces/list.go

```diff
@@ -68,7 +68,7 @@ func ListBuilder() *cobra.Command {
 	opts := new(ListOpts)
 	cmd := &cobra.Command{
 		Use:     "list",
-		Short:   "List namespaces for collections experiencing slow queries",
+		Short:   "List namespaces for collections experiencing slow queries.",
```

Comment:
Is the "up to 20 namespaces" part still accurate? This comes from our API docs:
```suggestion
		Short:   "Retrieves up to 20 namespaces for collections experiencing slow queries for a specified host.",
		Long:    "Namespaces appear in the following format: {database}.{collection}.",
		Example: "$ mongocli atlas performanceAdvisor namespaces list --processName atlas-111ggi-shard-00-00.111xx.mongodb.net:27017",
```